src/app/(main)/profile/page.tsx

```diff
@@ -76,44 +80,68 @@ export default function Page() {
                 <p className="text text-sm text-gray-500 mb-2">
                   Change Password:
                 </p>
-                <Input
-                  type="password"
-                  placeholder="New Password"
-                  value={formData.password}
-                  onChange={(event) =>
-                    setFormData({
-                      ...formData,
-                      password: event.target.value,
-                    })
-                  }
-                />
+                <div>
+                  <Input
+                    right={
+                      <div
+                        className="bg-gray-200 hover:bg-gray-300 px-4 py-2 rounded-md shadow-sm"
+                        onClick={() => setShowPassword((prev) => !prev)}
+                      >
+                        <FontAwesomeIcon
+                          icon={showPassword ? faEye : faEyeSlash}
+                          style={{ color: "#2d2a2a" }}
+                        />
+                      </div>
+                    }
+                    type={showPassword ? "text" : "password"}
+                    placeholder="New Password"
+                    value={formData.password}
+                    onChange={(event) =>
+                      setFormData({
+                        ...formData,
+                        password: event.target.value,
+                      })
+                    }
+                  />
+                </div>
               </div>
               <div>
-                <Input
-                  type="password"
-                  placeholder="Confirm Password"
-                  value={formData.confirm_password}
-                  onChange={(event) =>
-                    setFormData({
-                      ...formData,
-                      confirm_password: event.target.value,
-                    })
-                  }
-                />
+                <div className="relative z-0">
+                  <Input
+                    right={
+                      <div
+                        className="bg-gray-200 hover:bg-gray-300 px-4 py-2 rounded-md shadow-sm "
+                        onClick={() => setShowConfirmPassword((prev) => !prev)}
+                      >
+                        <FontAwesomeIcon
```

Comment:
No need for the font awesome library! Look at icons implemented elsewhere in the codebase, a simple `<i className="far fa-eye"/>` is all you need 😀